mailbox/store/src/test/resources/eml/invalidCharset.eml

```diff
@@ -0,0 +1,201 @@
+Return-Path: <aduprat@linagora.com>
```

Comment:
can we use a mail that as small as possible and still demonstrating the problem?